### ISSUE TITLE
Naming conventions for multiple image basemaps

### ISSUE THREAD

**Issue by mwloewen** [StraboSpot/StraboSpot2#586]
**Is your feature request related to a problem? Please describe.**
The naming conventions options to customize spot naming when adding spots to an image basemap, by prepending the map spot as a prefix, work GREAT for a single image. If you have to continue adding spots on a second (or third) image basemap, however, the numbering restarts at 1 for each new image. 

**Describe the solution you'd like**
I would prefer the numbering system to continue even if added onto multiple image basemaps. So if the last number was 12 on an image basemap, and you add a new spot to a new image (nested under the same map spot), the first number would be 13. 

**Describe alternatives you've considered**
Currently, we just have to manually keep track of an advance the spot ID on subsequent image basemaps. MOST of the time all spots can be on a single image, and the manual override is simply a step back to the situation before image basemap custom spot IDs naming conventions. Not a deal breaker.

**Comment by jessicagood** [comment]
Should I remove the option to 'Restart Numbering at 1 for Each Image Basemap/Strat Section' in favor of continuing the numbering throughout all of the image basemaps in a single parent Spot? Or have both options?

What should this new option be called (without having it be too wordy)? I haven't come up with anything good yet.
Restart Numbering at 1 for Spots on Image Basemaps/Strat Sections?
Continue Numbering for all Children Spots of a Single Parent Spot?
Continue Numbering for all Nested Spots of a Single Parent Spot?

What about the grandchildren and great grand children? Continuing the numbering for all of the image basemap Spots below a single parent could start to get unwieldly.

**Comment by jessicagood** [comment]
To recap: Change the 'Restart Numbering at 1 for Each Image Basemap/Strat Section' so it doesn't restart at 1 for each image basemap/strat section within a parent spot but keeps incrementing until we get to an image basemap/strat section in a NEW parent Spot. Then it starts over at 1 again. (This is all only applicable to names of spots that are on image basemaps or strat sections.) Is that right?

So then we'd need to change the line 'Restart Numbering at 1 for Each Image Basemap/Strat Section' to be something like
'Restart Numbering at 1 for 1st Child Spot across all Image Basemaps/Strat Sections within a Parent Spot' but that is awful wording. Any ideas on making this clearer?

**Comment by ClairePMartin** [comment]
@jessicagood @mwloewen maybe there are options: (like the new D/DD drop down) 

**Parent/Child Spot Naming Conventions**
_Choose how child spots are numbered across nested layers._

**1) Restart at 1**
Child spots start at 1 within each nested level.

**2) Continue Numbering**
Child spots are numbered consecutively across all nesting levels.

**Comment by jessicagood** [comment]
I'd like to keep a toggle, like the rest of the options.

Default would be continuous numbering across all Spots. 
The toggle would be changed to 'Start at 1 within each Nested Level'

**Comment by mwloewen** [comment]
"To recap: Change the 'Restart Numbering at 1 for Each Image Basemap/Strat Section' so it doesn't restart at 1 for each image basemap/strat section within a parent spot but keeps incrementing until we get to an image basemap/strat section in a NEW parent Spot. Then it starts over at 1 again. (This is all only applicable to names of spots that are on image basemaps or strat sections.) Is that right?" -- YES, that's how I imagine it.

For wording, I wonder if "Nested Level" might be confusing. What about:
"Restart Numbering at 1 for spots nested on Image Basemaps"
